src/Components/DislikeButton.js

```diff
@@ -16,7 +16,7 @@ export default function DislikeButton({ anime, variant }) {
   const size = variant === "contained" ? "large" : "medium";
 
   //Theme colors are taken from the default "palette" theme and MUST have a child color titled "main"
-  const buttonColor = disliked ? "primary" : "inherit";
+  let buttonColor = !selected ? "background" : disliked ? "primary" : "inherit";
```

Comment:
nit: this variable still won't change in its lifetime (when props change, re-render will run this function again, creating a new variable), so it can remain `const`.